tests/test_Source/test_source.py

```diff
@@ -181,6 +181,16 @@ def setup_method(self):
             lightcurve_time=np.linspace(-20, 50, 100),
             cosmo=cosmo,
         )
+        self.source6 = Source(
```

Comment:
Here, you have initiated a source class with roman filter and you need to test some functions. It would be great if you could test this class with test_ps_magnitude_no_variability() and test_ps_magnitude_with_variability() functions. These two functions are the most relavent functions for your changes.